### PR TITLE
Tickets/sitcom 1487

### PR DESCRIPTION
**Pull request by nsevilla**
Requesting merge of ticket to support vibration analysis

### CODE REVIEW

File: notebooks/tel_and_site/subsys_req_ver/m2/SITCOM-1487_TEA_vibration_analysis_MTM2.ipynb [hunk 1]
```diff
@@ -0,0 +1,257 @@
+{
```
Comment (Kelks19) [inline notebooks/tel_and_site/subsys_req_ver/m2/SITCOM-1487_TEA_vibration_analysis_MTM2.ipynb:1]: <p>So, Laura gave me this feedback and I am passing it to you! Perhaps you could write a markdown block with instructions that we have to uncomment the time windows of interest that we want the EFD data from </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/147/discussion/#comment-1832151626'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='4'/>

Comment (Kelks19) [inline notebooks/tel_and_site/subsys_req_ver/m2/SITCOM-1487_TEA_vibration_analysis_MTM2.ipynb:1]: <p>Could you add a small discussion/results block -- to summarise what we find from the analysis? This is for us when we are bringing all results together to write our grand technote! </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/147/discussion/#comment-1832151627'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='7'/>

Comment (Kelks19) [reply notebooks/tel_and_site/subsys_req_ver/m2/SITCOM-1487_TEA_vibration_analysis_MTM2.ipynb:1]: But smartly written notebook -- I learned a lot of tricks about how to pull EFD data smartly :D This notebook will be my reference!

Comment (nsevilla) [reply notebooks/tel_and_site/subsys_req_ver/m2/SITCOM-1487_TEA_vibration_analysis_MTM2.ipynb:1]: <p>OK, done!</p>

Comment (nsevilla) [reply notebooks/tel_and_site/subsys_req_ver/m2/SITCOM-1487_TEA_vibration_analysis_MTM2.ipynb:1]: <p>I added a small summary of the findings. I think further analysis and conclusions warrant looking at the whole picture.</p>

**Review by Kelks19 — APPROVED** [review]
Done !

**Pull request merged**